clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeleteExecutorGvcf.scala

```diff
@@ -123,35 +124,33 @@ class DeleteExecutorGvcf(deleteGvcf: DeleteGvcf) extends Executor {
           TransferGvcfV1Metadata(
             documentStatus = Option(DocumentStatus.Deleted),
             notes = gvcf.notes
-              .map(
-                notes => s"$notes\n${deleteGvcf.metadata.notes.getOrElse("")}"
-              )
-              .orElse(deleteGvcf.metadata.notes)
+              .map(notes => s"$notes\n${deleteGvcf.note}")
```

Comment:
Using `map` `orElse` this way seem confusing to me, is there a reason why `match { Some => ... None => ...}` is not used here?

Reply:
Just a style preference; I almost always reach for `.map`, `.orElse`, etc. so that it's clean to insert more chained operations in the middle / end.